src/utilities/asset-utils.ts

```diff
@@ -1,9 +1,12 @@
-declare const __webpack_public_path__: string;
-
-export function getAssetUrl(url: string) {
-  if (typeof __webpack_public_path__ === "undefined") {
-    return url;
-  }
-  const urlObj = new URL(url, __webpack_public_path__);
-  return urlObj.href;  
-}
+export const getAssetUrl = (path: string, unitCode: string, baseUrl: string, branch="main") => {
+  // This assumes images reside in a sub-directory of the unit directory.
+  const pathParts = path.split("/");
+  const filename = pathParts[pathParts.length - 1]; // e.g., "image.png"
```

Comment:
I'm not sure, but I think there might be some images in sub folders of images.
Can we maybe assume that these are always relative paths like `[unit]/[images]/[path]`? It seems like that might be true based the conversion happening in the image map. If that was the case then it seems you could just take that path relative to the baseUrl and you wouldn't have to process it at all.